README.md

```diff
@@ -89,6 +89,30 @@ ISAR_ROBOT_PACKAGE = isar_turtlebot
 ISAR_DEFAULT_MAP = turtleworld
 ```
 
+### Run with make
+Common commands for the project are in [Makefile](./Makefile).
+This requires the CLI program `make`.
+
+Usage:
+```bash
+make <command-in-makefile> # for example: make run
+```
+
+<details>
+<summary>Installation MacOS</summary>
+
+```bash
+brew install make
+```
+</details>
+<details>
+<summary>Installation Windows</summary>
+
+```bash
+choco install make
+```
+</details>
```

Comment:
Imo we dont need to mention how to install it on different operating systems, people can google that. Not a strong opinion, so if you want to keep it ok.